code/modules/mob/living/carbon/species.dm

```diff
@@ -962,4 +967,125 @@ var/global/list/whitelisted_species = list("Human")
 	brute_mod = 0.8
 	move_speed_multiplier = 2
 
-	blood_color = "#7FFF00"
\ No newline at end of file
+	blood_color = "#7FFF00"
+
+/datum/species/slime
+	name = "Slime"
+	icobase = 'icons/mob/human_races/r_slime.dmi'
+	deform = 'icons/mob/human_races/r_def_slime.dmi'
+	known_languages = list(LANGUAGE_SLIME)
+	meat_type = /obj/item/slime_heart
+	attack_verb = "glomps"
+
+	flags = IS_WHITELISTED | NO_BREATHE
+	anatomy_flags = NO_SKIN | NO_BLOOD | NO_BONES | NO_STRUCTURE | MULTICOLOR
+
+	spells = list(/spell/regen_limbs)
+
+	gender = NEUTER
+
+	tox_mod = 2
+
+	primitive = /mob/living/carbon/slime/pygmy
+
+	blood_color = "#96FFC5"
+	flesh_color = "#96FFC5"
+
+	cold_level_1 = 273  // Cold damage level 1 below this point.
```

Comment:
None of the cold levels on any of the other species in the file are formatted like that.
Should I change all of them?